### ISSUE TITLE
Screen reader is not announcing filter operation results

### ISSUE THREAD

**Issue by msftedad** [bryntum/support#11574]
**Test Environment:** 
OS: Windows 11 Enterprise Insider Preview Version 24H2(OS Build 26100.2892)
Browser: Microsoft Edge Version 132.0.2957.140, Chrome Version 133.0.6943.54
Screen readers: NVDA (2024.4.2), JAWS (Version 2025.2412.50) and Narrator.
URL: https://bryntum.com/products/schedulerpro/examples/frameworks/react-vite/highlight-resource-calendars/dist/

**User Impact:**
The lack of screen reader announcements for search results in the "Filter staff" field impacts blind and low-vision users, as well as individuals with cognitive disabilities who rely on auditory feedback, hindering their ability to navigate and use the search functionality effectively.

**Observation:** 
This issue is observed with all three screen readers.
Same issue is observed in chrome browser as well.

**Pre-Requisite:**

1. Turn on Screen Readers:
Narrator: Win + Shift + Enter
NVDA: Ctrl + Alt + N
JAWS: Ctrl + Alt + J

2. Verbosity:
Narrator: Default
JAWS: Beginner, Highest
NVDA: Default

**Repro Steps:**

1. Open the URL: https://bryntum.com/products/schedulerpro/examples/frameworks/react-vite/highlight-resource-calendars/dist/ in latest edge browser.
2. Press tab key to navigate to the 'Filter Staff' edit control present in the page.
3. Enter any data in the search edit field.
4. Now Verify whether Screen reader is announcing the search result information on for the 'Filter staff' search edit field in windows or not.

**Actual Result:**
Screen reader is not announcing the search result information on for the 'Filter staff' search edit field in windows. Screen reader is remaining silent on giving data in the search edit field.

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/7d50000f-9793-4d7f-85fb-6e03b26616f7" />

https://github.com/user-attachments/assets/377cc10d-7266-4d55-823b-b45c4f6f8679

**Refer Attachment:** 
1.Screen reader is not announcing the search result information on for the 'Filter staff' search edit field in windows..mp4
2.Screen reader is not announcing the search result information on for the 'Filter staff' search edit field in windows..png

**Expected Result:**
Screen reader should announce the search result information on for the 'Filter staff' search edit field in windows. 
Ex: Screen reader should announce the displayed search result as " 2 results found Henrik 1 of 1" and Screen reader should announce as "No records to display" on giving invalid data.
